trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Helper_Functions/windmilling_drag.py

```diff
@@ -68,24 +67,19 @@ def windmilling_drag(geometry,state):
                 break
 
     # getting geometric data from engine (estimating when not available)
-    for idx,network in enumerate(vehicle.networks):
-        try:
-            swet_nac = network.areas.wetted
-        except:
+    swet_nac = 0
+    for idx2,nacelle in enumerate(vehicle.nacelles):
+        D_nac = nacelle.diameter
+        if nacelle.length != 0.:
+            l_nac = nacelle.length
+        else:
             try:
-                D_nac = network.nacelle_diameter
-                if network.engine_length != 0.:
-                    l_nac = network.engine_length
-                else:
-                    try:
-                        MMO = vehicle.max_mach_operational
-                    except:
-                        MMO = 0.84
-                    D_nac_in = D_nac / Units.inches
-                    l_nac = (2.36 * D_nac_in - 0.01*(D_nac_in*MMO)**2) * Units.inches
-            except AttributeError:
-                print('Error calculating windmilling drag. Engine dimensions missing.')
-            swet_nac = 5.62 * D_nac * l_nac
+                MMO = vehicle.max_mach_operational
+            except:
+                MMO = 0.84
+            D_nac_in = D_nac / Units.inches
+            l_nac = (2.36 * D_nac_in - 0.01*(D_nac_in*MMO)**2) * Units.inches
+        swet_nac += 5.62 * D_nac * l_nac
```

Comment:
don't the nacelles already have a wetted area? Why aren't those used?

Then MMO can be removed and the issue resolved.